pkg/util/span/frontier.go

```diff
@@ -106,27 +106,28 @@ type Operation func(roachpb.Span, hlc.Timestamp) (done OpResult)
 var useBtreeFrontier = envutil.EnvOrDefaultBool("COCKROACH_BTREE_SPAN_FRONTIER_ENABLED",
 	metamorphic.ConstantWithTestBool("COCKROACH_BTREE_SPAN_FRONTIER_ENABLED", true))
 
-func enableBtreeFrontier(enabled bool) func() {
+// EnableBtreeFrontier sets useBtreeFrontier for testing purposes.
+func EnableBtreeFrontier(enabled bool) func() {
 	old := useBtreeFrontier
 	useBtreeFrontier = enabled
 	return func() {
 		useBtreeFrontier = old
 	}
 }
 
-func newBTreeFrontier() Frontier {
+func newBtreeFrontier() Frontier {
```

Comment:
i actually think BTree is more accurate than Btree... but it's probably not worth renaming everyone else.

Reply:
Yeah this was the path of least resistance to make it consistent lol